vrclient_x64/vrclient_x64/vrclient_main.c

```diff
@@ -891,6 +1008,49 @@ static EVRCompositorError ivrcompositor_submit_vulkan(
     return cpp_func(linux_side, eye, tex, bounds, flags);
 }
 
+EVROverlayError ivroverlay_set_overlay_texture(
+		EVROverlayError (*cpp_func)(void *, VROverlayHandle_t, Texture_t *),
+		void *linux_side, VROverlayHandle_t overlayHandle, Texture_t *texture,
+		unsigned int version, struct overlay_data *user_data)
+{
+	IUnknown *texture_iface;
+	HRESULT hr;
+
+	TRACE("%p, overlayHandle = %#x, texture = %p\n", linux_side, overlayHandle, texture);
+
+	TRACE("texture->eType = [%d] %s\n", texture->eType, vrclient_texture_type_name(texture->eType));
+
+	switch (texture->eType)
+	{
+		case TextureType_DirectX:
+			TRACE("D3D11\n");
+			texture_iface = texture->handle;
+			TRACE("texture_iface = %p\n", texture_iface);
+			if (texture_iface != NULL) {
+				TRACE("texture_iface->lpVtbl = %p\n", texture_iface->lpVtbl);
+			} else {
+				goto overlay_warn_out;
+			}
+
+#ifdef VRCLIENT_HAVE_DXVK
```

Comment:
Is there a wined3d code path? (do we want/need one at this point?)

Reply:
Oops, you’re totally right I missed that one. I’ll add it. Thanks!